### PR TITLE
ENH: move flow/pressure calls to the optics library

### PR DESCRIPTION
**Pull request by nrwslac**
## Description

- move cooling readbacks to optics library
- upgrade motion 2.0.1 -> 4.0.5
- upgrade optics 0.6.1 -> 0.7.0
- change G_PI lag monitoring 3urad -> 10urad to try and stop errors

## Motivation and Context

- implement alarm thresholds for NALMS

## How Has This Been Tested?

ran on PLC.

## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: lcls-plc-rixs-optics/_Config/NC/Axes/g_pi.xti [hunk 1]
```diff
@@ -1440,7 +1440,7 @@ External Setpoint Generation:
 		</Drive>
 		<Controller Name="Ctrl" CtrType="2">
 			<CtrPara PriorControlFactor="1">
-				<PosDiffControl Range="3" Time="0.2"/>
+				<PosDiffControl Range="10" Time="0.2"/>
```
Comment (ZLLentz) [inline lcls-plc-rixs-optics/_Config/NC/Axes/g_pi.xti:1443]: This is 10 urad?

File: lcls-plc-rixs-optics/rixs_optics/rixs_optics.plcproj [hunk 2]
```diff
@@ -249,19 +253,19 @@
     </PlaceholderReference>
   </ItemGroup>
   <ItemGroup>
-    <LibraryReference Include="lcls-twincat-motion,2.0.1,SLAC">
+    <LibraryReference Include="lcls-twincat-motion,4.0.5,SLAC">
```
Comment (ZLLentz) [inline lcls-plc-rixs-optics/rixs_optics/rixs_optics.plcproj:256]: That's quite the version bump

**Review by ZLLentz — APPROVED** [review]
This looks good based on the optics library PR (should have the same PV names as before, just using the library)

File: lcls-plc-rixs-optics/rixs_optics/rixs_optics.plcproj (hunk 2 above)
Comment (nrwslac) [reply lcls-plc-rixs-optics/rixs_optics/rixs_optics.plcproj:256]: trying to keep optics up to date :running_man:

File: lcls-plc-rixs-optics/_Config/NC/Axes/g_pi.xti (hunk 1 above)
Comment (ZLLentz) [reply lcls-plc-rixs-optics/_Config/NC/Axes/g_pi.xti:1443]: just making sure it's 10urad and not some grossly large physical quantity 👍

**Pull request merged**